bigbluebutton-html5/imports/ui/components/video-provider/hooks/index.ts

```diff
@@ -78,19 +78,19 @@ export const useStreams = () => {
       logger.warn({
         logCode: 'missing_stream_id',
         extraInfo: {
-          userId: user.userId,
-          role: user.role,
-          clientType: user.clientType,
+          userId: user?.userId || '',
```

Comment:
For preventing spurious top-level exceptions, this seems fine.
However, I'm wondering if we know the root cause - are these re-connection scenarios?
I'm also wondering if there's a better way to actually _handle_ this. Most of the props being ignored here are kind of essential - streamId being one of them.
If a stream hasn't got the minimal required fields, it should not be mapped at all but discarded - otherwise we're risking unexpected behavior.